### PR TITLE
added a simple printing of the key/button pressed

### PR DESCRIPTION
**Pull request by BielBdeLuna**
added a simple helper print method to know what key/button is being pressed, controlled via a bool cvar "in_namePressed" 1 or 0, I tested it and it works with keyboard, mouse and gamepad.

### CODE REVIEW

File: neo/framework/KeyInput.h [hunk 1]
```diff
@@ -312,6 +312,7 @@ class idKeyInput {
 	static const char *	BindingFromKey( const char *key );
 	static bool			KeyIsBoundTo( int keyNum, const char *binding );
 	static void			WriteBindings( idFile *f );
+        static void                     KeyReveal( int keyNum );
```
Comment (DanielGibson) [inline neo/framework/KeyInput.h:315]: please indenet with a tab, like the lines above

File: neo/framework/KeyInput.cpp [hunk 2]
```diff
@@ -775,6 +781,18 @@ void idKeyInput::PreliminaryKeyEvent( int keynum, bool down ) {
...
+/*
+===================
+idKeyInput::KeyReveal
+
+simple print in the console the name of the key pressed
+===================
+*/
+void idKeyInput::KeyReveal( int keyNum ) {
+    const char* keyName = KeyNumToString( keyNum, false );
+    common->Printf( "pressed the \"%s\"key.\n", keyName );
```
Comment (DanielGibson) [inline neo/framework/KeyInput.cpp:793]: please add a space between `\"%s\"` and `key`

**Comment by DanielGibson** [comment]
good idea, apart from the small issues I commented on this looks good to me :)

**Comment by BielBdeLuna** [comment]
now done, i made the requested changes changes :)

**Comment by DanielGibson** [comment]
merged, thank you! :)

**Pull request merged**